packages/steps/src/LionSteps.js

```diff
@@ -55,6 +55,12 @@ export class LionSteps extends ObserverMixin(LionLitElement) {
     ];
   }
 
+  get steps() {
+    let steps = this.children;
+    steps = Array.from(steps).filter(step => step.nodeName === 'LION-STEP');
+    return steps;
```

Comment:
taking into account [my explanation here](https://github.com/ing-bank/lion/pull/67#issuecomment-495569853), I would suggest you to change this to using [assignedNodes](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSlotElement/assignedNodes) on the default slot, because this is a safe assumption that we will have only `<lion-step>` elements in it